File: examples/museum/oseg_generated/csharp/GetMuseumHours_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class GetMuseumHoursDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var apiCaller = new OperationsApi(config);

            var response = apiCaller.GetMuseumHours(
                startDate: "2023-02-23",
                page: 2,
                limit: 15
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Operations#GetMuseumHours: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/museum/oseg_generated/java/GetMuseumHours_default_example.java
package org.openapimuseum.client.examples;

import org.openapimuseum.client.ApiException;
import org.openapimuseum.client.Configuration;
import org.openapimuseum.client.api.*;
import org.openapimuseum.client.auth.*;
import org.openapimuseum.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.List;
import java.util.Map;

public class GetMuseumHours_default_example
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();

        try
        {
            var response = new OperationsApi(config).getMuseumHours(
                LocalDate.parse("2023-02-23"),
                2,
                15
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling Operations#getMuseumHours");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}
